config/locales/work.de.yml

```diff
@@ -0,0 +1,8 @@
+de:
+  hyrax:
+    icons:
+      work:     'fa fa-file-text-o'
+    select_type:
+      work:
+        name:               "Work"
+        description:        "Work Werke"
```

Comment:
You may want to borrow values from Hyku for these (and rework them a bit to remove the 'generic' part).